tests/integrations/raftstore/test_split_region.rs

```diff
@@ -145,7 +145,15 @@ fn test_server_split_region_twice() {
     rx1.recv_timeout(Duration::from_secs(5)).unwrap();
 }
 
-fn test_auto_split_region<T: Simulator>(cluster: &mut Cluster<T>) {
+#[test_case(test_raftstore::new_node_cluster)]
+#[test_case(test_raftstore::new_server_cluster)]
+#[test_case(test_raftstore::new_incompatible_node_cluster)]
```

Comment:
Could you depict the difference among node_cluster, server_cluster and incompatible_node_cluster in comment of somewhere

Reply:
The modification here is just reuse v1's relevant tests for v2.

Reply:
Fine, I will add them.